### ISSUE TITLE
Missing modular record found at 2024-02-19T08:45:30.946738003Z

### ISSUE THREAD

**Issue by mhess-swl** [hashgraph/hedera-services#11858]
### Description

Hashscan transaction [here](https://hashscan.io/mainnet/transaction/1708332330.946738003)

Transaction with the following transaction ID is missing from the modular record stream:
```
transactionValidStart {
  seconds: 1708332316
  nanos: 902522561
}
accountID {
  accountNum: 1091756
}
```

### Steps to reproduce

Re-run events from round 162913624

### Hedera network

other

### Version

11805-diff-testing branch (with reverted commit 6954892)

### Operating system

macOS

**Comment by povolev15** [comment]
Latest run today these are the issues and just 77
Mismatched field counts  (expected but not find [scheduledTransactionID]) between expected status: SUCCESS
Mismatched field counts  (expected but not find [tokenTransferLists]) between expected receipt {
Mismatched field counts  (expected but not find [transactionFee]) between expected receipt {
Mismatched values, expected '4716962', got '4716961' - Matching field 'accountNum' Matching field 'accountID' Matching field 'receipt'  ==> expected: <4716962> but was: <4716961>
Mismatched values, expected '4716963', got '4716962' - Matching field 'accountNum' Matching field 'accountID' Matching field 'receipt'  ==> expected: <4716963> but was: <4716962>
Mismatched values, expected '4716964', got '4716963' - Matching field 'accountNum' Matching field 'accountID' Matching field 'receipt'  ==> expected: <4716964> but was: <4716963>
Mismatched values, expected '4716965', got '4716964' - Matching field 'accountNum' Matching field 'accountID' Matching field 'receipt'  ==> expected: <4716965> but was: <4716964>
Mismatched values, expected '4716966', got '4716965' - Matching field 'accountNum' Matching field 'accountID' Matching field 'receipt'  ==> expected: <4716966> but was: <4716965>
Mismatched values, expected '4716967', got '4716966' - Matching field 'accountNum' Matching field 'accountID' Matching field 'receipt'  ==> expected: <4716967> but was: <4716966>
Mismatched values, expected '4716968', got '4716967' - Matching field 'accountNum' Matching field 'accountID' Matching field 'receipt'  ==> expected: <4716968> but was: <4716967>
Mismatched values, expected '4716969', got '4716968' - Matching field 'accountNum' Matching field 'accountID' Matching field 'receipt'  ==> expected: <4716969> but was: <4716968>
Mismatched values, expected '4716970', got '4716969' - Matching field 'accountNum' Matching field 'accountID' Matching field 'receipt'  ==> expected: <4716970> but was: <4716969>

**Comment by mhess-swl** [comment]
Looks fixed by `11880-prioritize-synth-contract-creation` (commit `e4a64ee`)

**Comment by mhess-swl** [comment]
Fixed in `release/0.48` (commit `a393e24`)
